src/minizip/unzip.c

```diff
@@ -1162,7 +1162,7 @@ extern int ZEXPORT unzGetCurrentFileInfo (unzFile file,
         pfile_info->internal_fa = file_info64.internal_fa;
         pfile_info->external_fa = file_info64.external_fa;
 
-        pfile_info->tmu_date = file_info64.tmu_date;
+        pfile_info->tmu_date = file_info64.tmu_date,
```

Comment:
This does not look right

Reply:
Nice catch, that's an upstream bug.

It'll compile and run fine, though, without any negative side effects or implications.

Reply:
@metsma Do you want me to fix this in this PR or is it enough to go with the update as-is?
I can send the typofix upstream, if you want.

Reply:
Revert this change please

Reply:
> Revert this change please

Sure, done.